commands.go

```diff
@@ -0,0 +1,746 @@
+	p := CreateDatabasePayload{
+		DatabaseName: dbName,
+	}
+	data, err := json.Marshal(&p)
+	must(err)
+	res := &RavenCommand{
+		Method:      http.MethodPut,
+		URLTemplate: url,
+		Data:        data,
+	}
+	return res
+}
+
+// DatabasePutResponse decribes server response for e.g. CreateDatabaseCommand
+// https://sourcegraph.com/github.com/ravendb/ravendb-jvm-client@v4.0/-/blob/src/main/java/net/ravendb/client/serverwide/operations/DatabasePutResult.java#L7
+type DatabasePutResponse struct {
+	RaftCommandIndex int      `json:"RaftCommandIndex"`
+	Name             string   `json:"Name"`
+	DatabaseTopology Topology `json:"Topology"`
+	NodesAddedTo     []string `json:"NodesAddedTo"`
+}
+
+// DatabaseTopology describes a topology of the database
+// https://sourcegraph.com/github.com/ravendb/ravendb-jvm-client@v4.0/-/blob/src/main/java/net/ravendb/client/serverwide/DatabaseTopology.java#L8
+type DatabaseTopology struct {
+	Members                  []string          `json:"Members"`
+	Promotables              []string          `json:"Promotables"`
+	Rehabs                   []string          `json:"Rehabs"`
+	PredefinedMentors        map[string]string `json:"PredefinedMentors"` // TODO: not present in JSON response from python test
+	DemotionReasons          map[string]string `json:"DemotionReasons"`
+	PromotablesStatus        map[string]string `json:"PromotablesStatus"`
+	ReplicationFactor        int               `json:"ReplicationFactor"`
+	DynamicNodesDistribution bool              `json:"DynamicNodesDistribution"`
+	Stamp                    LeaderStamp       `json:"Stamp"`
+}
+
+// LeaderStamp describes leader stamp
+// https://sourcegraph.com/github.com/ravendb/ravendb-jvm-client@v4.0/-/blob/src/main/java/net/ravendb/client/serverwide/LeaderStamp.java#L3
+type LeaderStamp struct {
+	Index        int `json:"Index"`
+	Term         int `json:"Term"`
+	LeadersTicks int `json:"LeadersTicks"`
+}
+
+// ExecuteCreateDatabaseCommand executes CreateDatabaseCommand
+func ExecuteCreateDatabaseCommand(exec CommandExecutorFunc, cmd *RavenCommand) (*DatabasePutResponse, error) {
+	var res DatabasePutResponse
+	err := excuteCmdAndJSONDecode(exec, cmd, &res)
+	if err != nil {
+		return nil, err
+	}
+	return &res, nil
+}
+
+// DeleteDatabaseResult represents result of Delete Database command
+// https://sourcegraph.com/github.com/ravendb/ravendb-jvm-client@v4.0/-/blob/src/main/java/net/ravendb/client/serverwide/operations/DeleteDatabaseResult.java#L3
+type DeleteDatabaseResult struct {
+	RaftCommandIndex int `json:"RaftCommandIndex"`
+	//PendingDeletes   []string `json:"PendingDeletes"` // TODO: not send back in python test
+}
+
+// NewDeleteDatabaseCommand creates a new DeleteDatabaseCommand.
+// fromNode can be empty string
+// https://sourcegraph.com/github.com/ravendb/RavenDB-Python-Client@v4.0/-/blob/pyravendb/raven_operations/server_operations.py#L79
+// TODO: add timeToWaitForConfirmation time.Duration? Don't know the format,
```

Comment:
This uses .NET TimeSpan format 
https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings